doc/reference.md

```diff
@@ -301,6 +302,63 @@ import 'package:square_in_app_payments/in_app_payments.dart';
 ```
 --- 
 
+### startBuyerVerificationFlow
+
+Starts buyer verfication flow for card on file(cof). Displays verification view for some geographies. The method takes two callback parameters which correspond
```

Comment:
Reword:
Starts the buyer verification for a given payment source id. The most likely use case will be to pass in a card-on-file (cof). This will display a verification view to the user for some geographies to address Strong Customer Authentication.